desk/app/groups.hoon

```diff
@@ -2176,14 +2168,17 @@
   ++  ga-invite
     |=  =invite:g
     ^+  ga-core
+    ::  prevent spamming invites
+    ?:  ?=(~ vit.gang)  ga-core
+    ?:  (~(has by groups) p.invite)  ga-core
```

Comment:
The second one is obvious. The first one says "don't continue processing if we don't already have an invite marker"? That doesn't seem right, am I misunderstanding?

Reply:
```suggestion
    ::  prevent spamming invites
    ?.  ?=(~ vit.gang)  ga-core
    ?:  (~(has by groups) p.invite)  ga-core
```

oops this was a last minute addition, you are right